docs/src/Theory/Atmos/Microphysics_0M.md

```diff
@@ -0,0 +1,87 @@
+# Microphysics_0M
+
+The `Microphysics_0M.jl` module defines a 0-moment bulk parameterization of
+  moisture sink due to precipitation.
+It offers a simplified way of removing the excess water
+  without assuming anything about the size distributions of cloud
+  or precipitation particles.
+
+The ``q_{tot}`` (total water specific humidity) sink due to precipitation
+  is obtained by relaxation with a constant timescale
+  to a state with precipitable water removed.
```

Comment:
```suggestion
  to a state with condensate exceeding a threshold value removed.
```
(note that precipitable water usually, and misleadingly, is understood to include water vapor)